### ISSUE TITLE
Installation via Unity Package Manager fails for Version Number 0.17.6

### ISSUE THREAD

**Issue by DerMilchmann** [BioMotionLab/TUX#9]
Installation via Unity Package Manager fails for Version Number 0.17.6

`Cannot perform upm operation: Unable to add package [https://github.com/BioMotionLab/TUX.git#0.17.6]:
  [https://github.com/BioMotionLab/TUX.git#0.17.6] does not point to a valid package repository. No package manifest was found. Verify the repository URL and make sure the package is located in the root folder of the repository. [NotFound]`

Works fine with 0.17.5

**Comment by AdamBebko** [comment]
This is fixed in the next version coming out soon.

**Comment by DerMilchmann** [comment]
On a different issue, I recently created a new Project and ran into the misalignment problem again when using the default TextMeshPro Version (2.0.1). Updating to a recent Version (2.1.1) causes different misalignments.

Could you specify the correct preview Version (preview8) on the Installation page?

**Comment by AdamBebko** [comment]
Thanks again, I'll look into fixing this issue permanently using the non preview (release) version now that it's a stable release. Textmesh Pro previously did not save any alignment settings on prefabs that were saved through the package manager in prior versions, this seems to be fixed in the current version.

**Comment by AdamBebko** [comment]
@DerMilchmann I am pretty sure that this is still an internal bug in TextMeshPro that is still present even in Unity 2020.1.

Could you please try the following and let me know if it fixes your issue:

Using any non-preview version of TextmeshPro (eg: 2.1.1, 2.1.0 or the default one in Unity 2020.1):
    - In the unity editor, in your project, navigate to Packages/bmlTUX/Prefabs. 
    - Right click on ExperimentUI and click "Reimport". 

You can avoid this issue from arising entirely if you install the TextMeshPro Essentials BEFORE installing bmlTUX. It seems that the import of the essentials breaks any prefabs containing TMP elements in your project. I have updated the installation instructions accordingly.

I have a new release ready, will try to wait to hear back from you before making it live.
Thanks again for your reports!
Adam

**Comment by DerMilchmann** [comment]
I can confirm that I did not have any issues after intialising a new project with the following Versions:

- Unity 2020.1.2f1
- TextmeshPro 3.0.1 (default)
- bmlTUX 0.17.5

I created the project and imported the TMP Essentials before creating an experiment with the ScriptHelperTool. 
I could not test earlier TMP versions with as I wasn't able to downgrade the TMP with this version of Unity.

Trying out the Unity Version I am using right now, 2019.3.13f1, and importing the Essentials BEFORE installing bmlTUX, I had success using TMP version 2.1.1, but misalignments using 2.0.1 (default).
